main.go

```diff
@@ -16,7 +17,10 @@ type item struct {
 }
 
 func main() {
-	csvFile, err := os.Open("problems.csv")
+	csvPtr := flag.String("csv", "problems.csv", "a csv file in the format of 'question,answer'")
```

Comment:
What is a good name for this? You said that acronyms are upper cased but I didn't like `CSVPtr` and the variable name.

Reply:
`filepath` is fine.

Reply:
Because of the locality of the var you really don't need much more context.

Reply:
It is convention to call out that this is a pointer or is that a less meaningful implementation detail? I could see the helpfulness of distinguishing the difference if there are pointer values mixed with non-pointer values.